### DISCUSSION TITLE
Does Streamlink/Twitch GUI suddenly cause ban??

### DISCUSSION THREAD

**Discussion by YaliHorde** [streamlink/streamlink-twitch-gui#1002] [General]
So I got hit with a massive ban for 30 days due to botting or automation somehow...
First warning I have gotten, otherwise due to the high amount of ads on livestreams, I only watch VoDs now.

Also used the website to gain Drops on Brave browser on a game I played recently, not sure that should be bannable to not interact.
I recently used Twitch GUI, and I have no other logins or use of service violation...
Is there a chance these programs are causing ban suddenly?
![2024-03-18 231557](https://github.com/streamlink/streamlink-twitch-gui/assets/94868691/24e1d678-4434-4887-b997-35a230fee559)

Only thing I can come up with is that Twitch is adding a layer to make sure ads are being runned and not blocked.
When I check the reason that ban happens you can see it can be triggered by software or tools.
![2024-03-18 232840](https://github.com/streamlink/streamlink-twitch-gui/assets/94868691/8477d933-0104-4ca7-b846-89e87029ab4d)

**Comment by bastimeyer** [comment]
Streamlink Twitch GUI has nothing to do with watching streams, it's a Twitch stream browser using Twitch's public "helix" API, and a launcher for Streamlink which itself is responsible for downloading streams and feeding that data into a local video player.

Streamlink doesn't support official Twitch authentication anymore since many years, as you can read in its docs:
https://streamlink.github.io/cli/plugins/twitch.html

Whatever you're doing with your Twitch account and any kind of application is entirely up to you. Streamlink is just an HLS client and a stream URL resolver. If it was indeed responsible for causing accounts to be banned from regular usage, then you'd see other people posting, which clearly isn't the case.

Whatever you've done is unrelated to Streamlink and your ban is probably a justified action by Twitch.

**Reply by YaliHorde** [reply]
Well I don't know.
I haven't done anything out of the regular.
And I have been doing the same thing, watching streams and collecting loot from Drops.
So there arent any justifiable action to warrent ban, thats why I'm asking since this program is basically 3rd party software.
And Twitch is heavily pushing ads, and Streamlink Twitch GUI is blocking those or trying to.

**Comment by YaliHorde** [comment]
@bastimeyer 
As I suspected.. there is a token based issue that Twitch has banned me for.
And I haven't allowed any apps or integration of Twitch API/Token to anything in several months..
I recently changed my streamlink update, then launched a stream the other day, 10 hours before the ban........
Sooooooooooooooooo.. 
![2024-03-20 012734](https://github.com/streamlink/streamlink-twitch-gui/assets/94868691/51b45686-fdd7-44a1-b0cb-994d22f40d1e)
